docker-compose.yaml

```diff
@@ -30,17 +30,17 @@ services:
       - yoma-ga
       - host
     tty: true
-  ngrok-yoma-ga:
-    image: wernight/ngrok
-    env_file:
-      - environments/yoma-ga/aca-py-agent.default.env
-    command: ngrok http yoma-ga-agent:3020 --log stdout
-    networks:
-      - yoma-ga
-      - host
-    depends_on:
-      - ledger-nodes
-      - ledger-browser
+  # ngrok-yoma-ga:
+  #   image: wernight/ngrok
+  #   env_file:
+  #     - environments/yoma-ga/aca-py-agent.default.env
+  #   command: ngrok http yoma-ga-agent:3020 --log stdout
+  #   networks:
+  #     - yoma-ga
+  #     - host
+  #   depends_on:
+  #     - ledger-nodes
+  #     - ledger-browser
```

Comment:
Is this needed?

Reply:
no but we're not using this anyway. Thought we could free up some resources this way on the deploy machine. SHouldn't have snuck it in here I guess.